pkg/metaserver/application.go

```diff
@@ -0,0 +1,234 @@
+/*
+Copyright 2022 The KubeEdge Authors.
```

Comment:
```suggestion
Copyright 2021 The KubeEdge Authors.
```

Should be 2021 instead of 2022 here.
The file content was originally written in 2021 and the general rule is that the year to include in a copyright notice is the year of first publication of the work. 

Ref: https://www.copyrightlaws.com/copyright-symbol-notice-year/

> WHICH YEAR SHOULD YOU USE IN YOUR COPYRIGHT NOTICE?
> The general rule is that the year to include in a copyright notice is the year of first publication of the work. First publication is when the work is made available to the public without restriction. This includes selling a book or leaving free copies of it in a public place or posting content on your blog. Publication does not include, for example, circulating copies internally to co-workers or sending copies of a book to a publisher.
> 
> For compilations or derivative works which incorporate previously published content, the year of first publication of the compilation or derivative work is sufficient.

Reply:
done